packages/main/src/CardHeader.hbs

```diff
@@ -0,0 +1,34 @@
+<div
+	class="{{classes}}"
+	@click="{{_headerClick}}"
+	@keydown="{{_headerKeydown}}"
+	@keyup="{{_headerKeyup}}"
+	role="{{ariaHeaderRole}}"
+	aria-labelledby="{{ariaLabelledByHeader}}"
+	aria-level="{{ariaLevel}}"
+	aria-roledescription="{{ariaCardHeaderRoleDescription}}"
+	tabindex="0"
+	id="{{_id}}--header"
+>
+	{{#if hasAvatar}}
+		<div id="{{_id}}-avatar" class="ui5-card-avatar" aria-label="{{ariaCardAvatarLabel}}">
```

Comment:
Some classes names should be renamed. From one side there is "ui5-card-header-text" class, from other: ui5-card-avatar, ui5-card-title, ui5-card-subtitle, ui5-card-status. As the component is called ui5-card-header, perhaps we should start using ui5-card-header-* prefix for the classes.